lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
```

Comment:
But having it as a class variable makes it explicit that the variable is specific to the whole class and not just one instance.